src/components/skills-quiz/SelectedJobSkills.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { useContext } from 'react';
+import { Badge } from '@edx/paragon';
+import { SKILL_NAME_CUTOFF_LIMIT } from './constants';
+import { SkillsContext } from './SkillsContextProvider';
+import { shortenString } from '../course/data/utils';
+import { ELLIPSIS_STR } from '../course/data/constants';
+
+const SelectedJobSkills = () => {
+  const { state } = useContext(SkillsContext);
+  const {
+    interestedJobs, selectedJob,
+  } = state;
+
+  const selectedJobDetails = interestedJobs.filter(job => job.name === selectedJob);
+  let selectedJobSkills = selectedJobDetails[0]?.skills?.sort((a, b) => (
+    (a.significance < b.significance) ? 1 : -1));
+  selectedJobSkills = selectedJobSkills?.slice(0, 5);
+
+  return (
+    <div style={{ paddingLeft: '10%' }} className="mb-3 mt-5">
+      <h4>Top Skills for {selectedJob}</h4>
+      <div>
+        {selectedJobSkills?.map(skill => (
+          <Badge
+            key={skill.name}
+            className="skill-badge"
+            variant="light"
+            data-testid="top-skills-badge"
+          >
+            { shortenString(skill.name, SKILL_NAME_CUTOFF_LIMIT, ELLIPSIS_STR) }
```

Comment:
This `SKILL_NAME_CUTOFF_LIMIT` was set according to the dimensions of the course about page, will it work for skills-quiz page too?

Reply:
Yes, it will, as we have to copy the same design logic to show the skills in this section as to skills in course card.

Reply:
I think we don't need to cut off the size and there is no spacing issue at this place. Removing this logic